tooling/nargo_cli/Cargo.toml

```diff
@@ -75,9 +81,13 @@ fm.workspace = true
 criterion.workspace = true
 pprof.workspace = true
 paste = "1.0.14"
+proptest.workspace = true
+sha2 = "0.10"
+sha3 = "0.10"
```

Comment:
We depend on these crates in the crate `blackbox_solver` so it would be good to make these workspace deps.

Reply:
Okay, I wasn't sure if it was deliberately hidden away in the blackbox solver. Moved into the workspace.